src/test/java/org/jpeek/skeleton/SkeletonTest.java

```diff
@@ -38,6 +39,7 @@
 public final class SkeletonTest {
 
     @Test
+    @Disabled
```

Comment:
@dgarus By disabling the test now, you have removed the existing testing functionality, right?

Reply:
Yes, for a while.

Reply:
@dgarus IMHO not a good idea to disable existing testing functionality. We need to think how we can achieve the same without disabling existing tests.

Reply:
Ok, I created a separate test.